layouts/partials/tw/main.html

```diff
@@ -30,225 +16,242 @@
+      {{ if .PublishDate.Before $eighteenMonthsAgo }}
+        <p
+          class="ml-1 inline-block bg-orange px-3 font-sans text-base font-semibold uppercase text-white"
         >
-          {{ .Title }}
-        </h1>
-
-        {{ with $params.byline }}
-          <h2
-            class="mt-1 font-sans text-base leading-tight tracking-normal text-g-8 sm:leading-normal"
-          >
-            {{ if $bylineLink }}
-              <a href="{{ $bylineLink }}">
-                {{- block "main-byline" $params }}
-                  <span class="font-semibold">
-                    by
-                    <span>
-                      {{- .byline -}}
-                    </span>
-                  </span>
-                  {{ if not .evergreen | and .published }}
-                    <span class="font-light">
-                      |
-                      <time datetime="{{ .publishedISO }}">
-                        {{- .published -}}
-                      </time>
-                    </span>
-                  {{ end }}
-                {{ end -}}
-              </a>
-            {{ else }}
-              {{ template "main-byline" $params }}
-            {{ end }}
-          </h2>
-        {{ end }}
+          From the archives
+          {{ .PublishDate.Format "2006" }}
+        </p>
+      {{ end }}
+    {{ end }}
 
 
-        <div class="mt-4">
-          {{ partial "tw/social-buttons.html" . }}
-        </div>
-      </header>
+    <h1
+      class="mt-1 text-2xl font-bold leading-tight tracking-tight text-black sm:text-4xl"
+    >
+      {{ .Title }}
+    </h1>
 
-      <div class="{{ $mainWellClass }} w-full print:!max-w-md">
-        {{ partial "tw/image-hero.html" . }}
-      </div>
+    {{ with $params.byline }}
+      <h2
+        class="mt-1 font-sans text-base leading-tight tracking-normal text-g-8 sm:leading-normal"
+      >
+        {{ if $bylineLink }}
+          <a href="{{ $bylineLink }}">
+            {{- block "main-byline" $params }}
+              <span class="font-semibold">
+                by
+                <span>
+                  {{- .byline -}}
+                </span>
+              </span>
+              {{ if not .evergreen | and .published }}
+                <span class="font-light">
+                  |
+                  <time datetime="{{ .publishedISO }}">
+                    {{- .published -}}
+                  </time>
+                </span>
+              {{ end }}
+            {{ end -}}
+          </a>
+        {{ else }}
+          {{ template "main-byline" $params }}
+        {{ end }}
+      </h2>
+    {{ end }}
 
-      <div class="{{ $mainWellClass }}">
-        <div x-data x-intersect.once="$dispatch('open-sticky')"></div>
-        {{ partial "tw/content-well.html" . }}
+  </header>
+  <article
+    class="mx-auto flex max-w-screen-md flex-wrap md:justify-center md:gap-x-12 lg:col-span-8 lg:justify-start lg:gap-x-8"
+  >
+    <div class="w-full print:!max-w-md">
```

Comment:
I don't think we need `print:!max-w-md` anymore.

Let's add a todo item to check the print styles in a follow up PR.